src/wp-includes/ms-site.php

```diff
@@ -977,7 +987,7 @@ function clean_blog_cache( $blog ) {
 	 *
 	 * @since 4.6.0
 	 *
-	 * @param string  $id              Site ID as a numeric string.
+	 * @param string  $id Site ID as a numeric string.
```

Comment:
This change is not related to this ticket, and appears to not follow the documentation standards.